lib/services/local/users.go

```diff
@@ -380,6 +380,9 @@ func (s *IdentityService) DeleteOIDCConnector(connectorID string) error {
 func (s *IdentityService) GetOIDCConnector(id string, withSecrets bool) (*services.OIDCConnector, error) {
 	out, err := s.backend.GetVal(connectorsPath, id)
 	if err != nil {
+		if teleport.IsNotFound(err) {
+			return nil, trace.Wrap(teleport.NotFound(fmt.Sprintf("OpenID Connect connector '%v' not found", id)))
```

Comment:
Is this a user-facing error? "Connector not found" does not tell me what to do. What is the most likely cause for seeing this? Configuration error? Perhaps we should say "OpenID connector 'xxx' is not supported?"